### DISCUSSION TITLE
Vim <esc> was not working due to Vimium plugin

### DISCUSSION THREAD

**Discussion by avi-cenna** [typst/webapp-issues#176] [General]
Hello,

Just wanted to put this out there in case anyone else runs into this. I enabled the Vim mode in the Typst web app, and noticed that all the escape keys `esc` and `C-[` did not work. They just removed focus from the editor instead of getting back into Normal mode.

However, I soon realized that it was because I am using the Vimium plugin for Chrome, which allows Vim keybindings for controlling Chrome. I simply had to disable Vimium on the Typst web app and everything started working.

P.S. thanks for your work on Typst! I have some experience with LaTeX and this is a really cool project.

**Comment by rwmpelstilzchen** [comment]
Same thing with [Tridactyl](https://tridactyl.xyz/).
